MykytaTitar/HW9/First_GitHub_Task.py

```diff
@@ -0,0 +1,33 @@
+import random
```

Comment:
good job
look at this implementation

user_name = input('Hello! What is your name?\n')

print(f'Well, {user_name}, I am thinking of a number between 1 and 20.')


while guesses_counter < 10:

    guess_number = int(input('Take a guess! Enter your number: '))

    guesses_counter = guesses_counter + 1

    if guess_number == number:
        print(f'Good job, {user_name}! You are a winner!!!You guessed my number in {guesses_counter} guesses!')
        break
    
    elif 1 <= guess_number <= 20 and guess_number < number:
        print('Your number is less.') 

    elif 1 <= guess_number <= 20 and guess_number > number:
        print('Your number is more.')

    elif not 1 <=guess_number <= 20:
        print(f"Your number is not in the range 1 to 20. Try again:)")
    
       
else:
    print(f'Nope. The number I was thinking of was {number}. You used {guesses_counter} attempts.')